src/nunavut/cli/__init__.py

```diff
@@ -115,7 +115,7 @@ def _make_parser() -> argparse.ArgumentParser:
 
         Additional directories can also be specified through the environment variable
         UAVCAN_DSDL_INCLUDE_PATH, where the path entries are
```

Comment:
```suggestion
        DSDL_INCLUDE_PATH, where the path entries are
```

I suppose there is no point documenting support for the legacy variable?

Reply:
Fixed.